lti_consumer/templates/html/student.html

```diff
@@ -1,18 +1,23 @@
-<h2 class="problem-header">
-    ## Translators:  "External resource" means that this learning module is hosted on a platform external to the edX LMS
-    ${display_name} (External resource)
-</h2>
+% if display_header:
+    <h2 class="problem-header">
+        ## Translators:  "External resource" means that this learning module is hosted on a platform external to the edX LMS
+        ${display_name} (External resource)
+    </h2>
 
-% if has_score and weight:
-    <div class="problem-progress">
-        % if module_score is not None:
-            (${"{points} / {total_points} points".format(points=module_score, total_points=weight)})
-        % else:
-            (${"{total_points} points possible".format(total_points=weight)})
-        % endif
-    </div>
+    % if has_score and weight:
+        <div class="problem-progress">
+            % if module_score is not None:
+                (${"{points} / {total_points} points".format(points=module_score, total_points=weight)})
```

Comment:
@robrap I guess I could be a "good citizen" and address a few prior existing i18n/safe template issues. If something gets overly burdensome, I'll bring it up.